server/routes/scheduledJobs.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express'
+import scheduledJobsController from '../controllers/scheduledJobsController.js';
+
+const router = Router()
+
+// Schedules the reminder to return the dish
+// Requires transId, userId, dishId as query params
+router.get('/dishCheckout', async (req, res, next) => {
```

Comment:
We shouldn't be triggering the reminder, we should be running a job every N-days to run a query on all outdated transactions